src/main/resources/defaultConfig.yml

```diff
@@ -91,6 +91,13 @@ message-prefix: §7§l>>
 info-colour: §a
 error-colour: §c
 
+# -----------------------------------------------------
+# | Shortlink: https://shlink.io/
+# -----------------------------------------------------
+shortlink:
+  enable: false
+  apikey: yourapi-key
+  host: https://your.shortlink.host
 
 # NOTE: Do not change
 config-version: 1.3
```

Comment:
Config version needs to be updated to **1.4**
Please also update it here https://github.com/AlpsBTE/Plot-System/blob/f9779bf8b44a341fc6d8a811b074945da89bb42b/src/main/java/com/alpsbte/plotsystem/core/config/ConfigManager.java#L47